collect_app/src/main/java/org/odk/collect/android/database/ODKSQLiteOpenHelper.java

```diff
@@ -144,6 +146,7 @@ public synchronized SQLiteDatabase getWritableDatabase() {
                     try {
                         mDatabase.close();
                     } catch (Exception e) {
+                        Timber.e(e, "Unable to close database due to: %s", e.getMessage());
```

Comment:
The exception message always gets logged when you pass in an exception so you don't need to add it to your message. If you'd like to add a message here, "Unable to close database" is enough. Same comment for the two logs below.